sdks/python/tests/e2e/test_prompt.py

```diff
@@ -159,3 +159,72 @@ def test_prompt__format():
     assert result == "Hello, John from The Earth! Nice to meet you, John."
 
     assert prompt.prompt == template
+
+
+def test_prompt__create_with_default_type(opik_client: opik.Opik):
+    unique_identifier = str(uuid.uuid4())[-6:]
+
+    prompt_name = f"some-prompt-name-{unique_identifier}"
+    prompt_template = f"some-prompt-text-{unique_identifier}"
+
+    prompt = opik_client.create_prompt(
+        name=prompt_name,
+        prompt=prompt_template,
+    )
+
+    assert prompt.type == "mustache"  # Verify default type
```

Comment:
Let's just add this line to `test_prompt__create__happyflow` and remove this test because it is actually almost the same.